From 12817ac3db28b1945a664ea3411186e1e64f6ed4 Mon Sep 17 00:00:00 2001
From: Victor Hermann Chiletto <v@hnn.net.br>
Date: Tue, 9 Jun 2020 01:35:53 -0300
Subject: [PATCH 52/54] winegstreamer: Fix unrecognized format errors when
 transforming caps to media type

Probably not needed, but it's nice to get this out of the way
---
 dlls/winegstreamer/mfplat.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/dlls/winegstreamer/mfplat.c b/dlls/winegstreamer/mfplat.c
index bcfcd398ec..39b98f3eba 100644
--- a/dlls/winegstreamer/mfplat.c
+++ b/dlls/winegstreamer/mfplat.c
@@ -910,6 +910,8 @@ static IMFMediaType* transform_to_media_type(GstCaps *caps)
             {
                 if (!(strcmp(format, "WVC1")))
                     IMFMediaType_SetGUID(media_type, &MF_MT_SUBTYPE, &MFVideoFormat_WVC1);
+                else if (!(strcmp(format, "WMV3")))
+                    IMFMediaType_SetGUID(media_type, &MF_MT_SUBTYPE, &MFVideoFormat_WMV3);
                 else
                     FIXME("Unrecognized format %s\n", format);
             }
-- 
2.28.0

